lib/procinfo.h

```diff
@@ -82,4 +82,10 @@ extern void procinfo_non_boinc(PROCINFO&, PROC_MAP&);
 extern double process_tree_cpu_time(int pid);
     // get the CPU time of the given process and its descendants
 
+extern double total_cpu_time();
+    // total CPU time, as reported by OS
```

Comment:
The comment in `procinfo_unix.cpp` more accurately states that this returns *user-mode* CPU time. Both comments are missing the returned unit, which is, I believe, seconds.

Reply:
fixed.  Time in BOINC are always seconds unless otherwise specified